schema/url/standalone-map/plugin-floating-lineplot.raml

```diff
@@ -0,0 +1,55 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+
+  FloatingLineplotPostRequest:
+    type: VisualizationRequestBase
+    properties:
+      config: FloatingLineplotSpec
+
+  FloatingLineplotSpec:
+    additionalProperties: false
+    properties:
+      outputEntityId:
+        type: string
+      xAxisVariable: 
+        type: VariableSpec
+      yAxisVariable: 
+        type: VariableSpec
+      yAxisNumeratorValues:
+        type: string[]
+        required: false
+      yAxisDenominatorValues:
+        type: string[]
+        required: false
+      overlayVariable: 
+        type: VariableSpec
+        required: false
+      overlayValues?:
+        type: BinRange[] # categoricals can just ignore bin start and end
+      binSpec:
+        type: BinSpec
+        required: false
+      valueSpec:
+        type: string
+        enum: ['median','mean','geometricMean','proportion']
+      # for now assume 95 ci only, eventually values like 'se', 'sd', 'ci' etc
+      errorBars:
+        type: StringBoolean
+      viewport:
+        type: NumericViewport
+        required: false
+
+  FloatingLineplot:
+    additionalProperties: false
+    properties:
+      data:
+        type: LineplotData[]
```

Comment:
Are we OK using `LineplotData` as the response type, even though it will never contain (the optional) `facetVariableDetails`? I think so, but just wanted to raise this.